src/crimson/osd/shard_services.cc

```diff
@@ -348,6 +351,10 @@ void OSDSingletonState::handle_conf_change(
   if (changed.count("osd_max_trimming_pgs")) {
     snap_reserver.set_max(conf->osd_max_trimming_pgs);
   }
+  if (changed.count("osd_beacon_report_interval")) {
```

Comment:
This conf change handler should be on the OSD if the OSD owns the beacon_timer.

Reply:
Fixes, I added the OSD as an observer.